<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_DiodeTest</class>
 <widget class="QDialog" name="Dialog_DiodeTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Diode Test</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>11</y>
     <width>1011</width>
     <height>551</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>S.No</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Line X</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>LineY</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Min</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Measured Value</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Max</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Units</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Results</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButton_Measure">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>590</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Measure</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Save">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>590</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Abort">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>590</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Abort</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Back">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>590</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>590</y>
     <width>401</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_Units">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>590</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Units:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>590</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Ohms</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
